pkg/config/agent/config.go

```diff
@@ -424,3 +426,9 @@ type OVSBridgeConfig struct {
 	// Defaults to false.
 	EnableMulticastSnooping bool `yaml:"enableMulticastSnooping,omitempty"`
 }
+
+type HostNetworkAccelerationConfig struct {
+	// Enable to accelerate traffic on Node host networking. Currently only Pod-to-Pod traffic is supported, and more cases
+	// will be supported in the future.
```

Comment:
Do you have some examples what will be supported in future? And we will have a knob for each feature to be implemented with nftables? Will we have other settings in addition to the knobs?

Reply:
External-to-Service and Pod-to-external traffic will be supported in the future, but we don't plan to add specific flags for them. One more thing, we may introduce another integer option type to accelerate the connections having more than N packets.

Reply:
Nft can also accelerate `host network network` part of Service traffic path: external client -> Node network -> `host network network` -> OVS pipeline (noEncap/encap/hybrid).

Reply:
Ok, I am also thinking if we may leverage other mechanisms like eBPF in future to accelerate host traffic. But sure maybe for now we can just keep your current generic option definiton.